internal/voting/sql.go

```diff
@@ -4,192 +4,198 @@ package voting
 	sql.go contains information about the sql tables used by the voting system.
 */
 
+/*
+	Final schema after all the upgrades:
+	resolutions:
+		- id: uuid
+		- body: bytea
+		- type: bytea
+		- vote_body_proposer: bytea
+		- expiration: int8
+
+	resolution_types:
+		- id: uuid
+
+	voters:
+		- id: uuid
+		- name: bytea
+		- power: int8
+
+	votes:
+		- resolution_id: uuid
+		- voter_id: uuid
+
+	processed:
+		- id: uuid
+*/
```

Comment:
Thanks for laying this out as requested.  @brennanjl This was because I noted that the `CREATE TABLE` statement creates a potentially obsolete table definition, and all the queries written for the application at a newer table structure will be confusing or hard to write without an *effective* schema description.

I had mistakenly assumed there was a short circuit in `versioning.Upgrade` where it would break out of the version loop after applying the `initTables` step and set version to latest without applying all the upgrades to empty tables.  This is somewhat strange to do, but it should be alright if properly documented like this.

If that behavior isn't intended or desired, we can change it.

Reply:
Yeah, this is why I assumed that a -1 version would be the fastpath with the whole `CREATE TABLE` for the current schema.